docs/WritingTests.asciidoc

```diff
@@ -428,24 +428,21 @@ Children B, C, D can run and finish anytime, but A must run until all B, C, D fi
 
 === Test synchronization and locking API
 
-OpenQA provides locking server through lock API. To use lock API import +lockapi+ package (_use lockapi;_) in your test file.
-Lock API provides functions: +mutex_create+, +mutex_lock+, +mutex_unlock+, +mutex_wait+. Each of these functions take at least one parameter: name of the lock. Note that lock name can't contain "-" character.
-Locks are associated with caller's job - locks can't be unlocked by different job then the one who locked the lock.
+OpenQA provides a locking mechanism. To use it in your test files import the +lockapi+ package (_use lockapi;_). It provides the following functions: +mutex_create+, +mutex_lock+, +mutex_unlock+, +mutex_wait+
```

Comment:
can you explain why you against "locking API" ? for me it is perfect term here

Reply:
I was just replacing the "locking server" with "locking mechanism". I thought the "through lock API" provides no further value so I removed it.

By the way: The sentence as it was written in the first place definitely makes no sense which is the reason I wanted to rephrase it. The original sentence reads like the API would provide the server but actually the server provides the API.

Reply:
I am not suggesting to keep "throught lock API" , I am suggesting to replace "locking server through lock API" with just "locking API" or "lock API" ( not sure which one is better )

Reply:
I agree with Anton. How the API is implemented doesn't matter at this point